src/sidebar/components/Annotation/AnnotationEditor.tsx

```diff
@@ -169,6 +178,41 @@ function AnnotationEditor({
 
   const mentionsEnabled = store.isFeatureEnabled('at_mentions');
   const usersWhoAnnotated = store.usersWhoAnnotated();
+  const focusedGroupMembers = store.getFocusedGroupMembers();
+  const focusedGroupId = store.focusedGroupId();
+
+  // Track previous members loading and the group for which it was triggered.
+  // This allows to abort that loading if the group changes before finishing.
+  const prevMembersLoading = useRef<null | {
```

Comment:
This logic could be pushed down into the groups service. The only thing that needs to happen in the `AnnotationEditor` is the trigger to request that group members be fetched if they are not already available.

Placing this logic in the annotation editor creates an issue that a second request can be triggered if you close an annotation editor and open another one while in flight. Also if other components in the application needed to access members in the focused group they could end up with the same logic.